modules/labs-react/search-form/lib/SearchForm.tsx

```diff
@@ -464,8 +461,10 @@ export class SearchForm extends React.Component<SearchFormProps, SearchFormState
               clearButtonAriaLabel={clearButtonAriaLabel}
               labelId={labelId}
             >
-              <SearchInput
-                ref={this.inputRef}
+              <FormField.Input
+                as={SearchInput}
+                ref={this.inputRef as any}
+                cs={{maxWidth: grow ? '100%' : maxWidth}}
```

Comment:
Should we do this as a modifier or leave using the cs prop?

Reply:
Or would this be an API change

Reply:
leaving this as is since it's a labs component and hasn't been converted to a stencil

Reply:
This is part of labs, so we are not updating this